docs/Users_Guide/roc_diagram.rst

```diff
@@ -51,27 +52,103 @@ METplus Configuration
 **Default Configuration File**
 
 The following is the `mandatory`, roc_diagram_defaults.yaml configuration file,
-which serves as a good starting point for creating a ROC diagram
-plot as it represents the default values set in METviewer
+which serves as a starting point for creating a ROC diagram plot,  as it represents
+the default values set in METviewer.
+
+**NOTE**: This default configuration file is automatically loaded by roc_diagram.py.
 
 .. literalinclude:: ../../metplotpy/plots/config/roc_diagram_defaults.yaml
 
+
+
+Modify the `stat_input` setting (input data file) in this default config file to ensure that the data file is accessible.
```

Comment:
This whole section here:

> Modify the `stat_input` setting (input data file) in this default config file to ensure that the data file is accessible.
> Open the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/config/roc_diagram_defaults.yaml file and replace
> the relative path in the stat_input entry `stat_input:  ../../../test/roc_diagram/plot_20200507_074426.data`
> with the path to your $METPLOTPY_SOURCE directory.  For example, if your $METPLOTPY_SOURCE is /username/plotting,
> then the `stat_input` setting will look like the following:
> 
> `stat_input:  /username/plotting/METplotpy/test/roc_diagram/plot_20200507_074426.data`

is not needed for users on the HPC.  They will not be able to modify roc_diagram_defaults.yaml unless they copy this file over to their own working area, however, if they are running:
```
python /apps/contrib/MET/METplotpy/METplotpy-feature_153_intermed_files/metplotpy/plots/roc_diagram/roc_diagram.py custom_roc_diagram.yaml
```
They will not need to also modify roc_diagram_defaults.yaml because they will be able to run using the modified custom_roc_diagram.yaml file.  If I understand correctly, this custom_roc_diagram.yaml will override values in the roc_diagram_defaults.yaml.